java/README.md

```diff
@@ -81,17 +82,3 @@ verification tasks.
 Exceptions with well-defined behaviors can be explicitly granted if
 they allow a wider range of benchmarks to be included in the
 collection.
-
-## Properties
```

Comment:
What is going to replace this?

Reply:
I moved the description of the property to the top.